acct-service/README.md

```diff
@@ -181,9 +181,9 @@ Visit https://accounting.dev.ubirch.com/docs on your browser to see the swagger
 docker-compose up
 
 # run the evolutions scripts that are stored in 'src/main/resources/db/migrations'
-cd src/main/resources/db
```

Comment:
why is /db gone?

Reply:
`cassandra-migrate.yml` is in the `src/main/resources`directory.